docs/06_Oversikt_over_klasser.adoc

```diff
@@ -215,49 +267,52 @@
 | URI | dct:MediaType
 | Referanse | http://purl.org/dc/terms/MediaType
 | Status | Valgfri
-
 |===
-=== Tidsrom [[klasse-tidsom]]
+
+=== Proveniensbeskrivelse [[klasse-proveniensbeskrivelse]]
 
 [cols="30s,70d"]
 |===
-| Engelsk navn | Period of time
-| Beskrivelse | Et tidsintervall som er navngitt eller definert av en start- og sluttdato.
-| URI | dct:PeriodOfTime
-| Referanse | http://dublincore.org/documents/dcmi-terms/#terms-PeriodOfTime
+| Engelsk navn | Provenance Statement
+| Beskrivelse | En beskrivelse av enhver endring i eierskap og forvaltning av en ressurs (fra den ble opprettet) som har betydning for autentisitet, integritet og fortolkning.
+| URI | dct:ProvenanceStatement
+| Referanse | http://dublincore.org/documents/dcmi-terms/#terms-ProvenanceStatement
 | Status | Valgfri
 |===
 
-=== Utgivertype [[klasse-utgivertype]]
+
+=== Rettighetsutsagn [[klasse-rettighetsutsagn]]
 
 [cols="30s,70d"]
 |===
-| Engelsk navn | Publisher type
-| Beskrivelse | Type organisasjon som fungerer som en utgiver
-| URI | skos:Concept
-| Referanse | http://www.w3.org/TR/vocab-adms/#dcterms-type
+| Engelsk navn | Rights statement
+| Beskrivelse | En utsagn om immaterielle rettigheter knyttet til en ressurs, et juridisk dokument som gir offisiell tillatelse til å gjøre noe med en ressurs, eller en uttalelse om tilgangsrettigheter.
+| URI | dct:RightsStatement
+| Referanse | http://dublincore.org/documents/dcmi-terms/#terms-RightsStatement
 | Status | Valgfri
 |===
 
-=== Relasjon [[klasse-relasjon]]
+
+=== Sjekksum [[klasse-sjekksum]]
 
 [cols="30s,70d"]
 |===
-| Engelsk navn | Relationship
-| Beskrivelse | For å knytte ytterligere informasjon til en relasjon mellom ressurser
-| URI | dcat:Relationship
-| Referanse | https://www.w3.org/TR/vocab-dcat-2/#Class:Relationship
+| Engelsk navn | Checksum
+| Beskrivelse | En beskrivelse som muliggjør autentisering av en fil. Flere sjekksumtyper og kryptografiske algoritmer kan brukes.
+| URI | spdx:Checksum
+| Referanse | http://spdx.org/rdf/terms#Checksum
 | Status | Valgfri
+| Kommentar | Norsk utvidelse. Denne klassen er ikke eksplisitt tatt med i BRegDCAT-AP, men var med i DCAT-AP-NO v.1.1.
```

Comment:
Du fant ut at den faktisk var i bruk? :)

Reply:
Nei, ingen treff i de filene jeg fikk fra Stig, hverken på denne klassen Sjekksum (spdx:Checksum) eller på Utgivertype (skos:Concept) som begge blir "norsk utvidelse" ift. BRegDCAT-AP. Tenker i første omgang nå å ta med alt som var i DCAT-AP-NO v.1.1 (for ikke å blande kortene for mye).